### PR TITLE
Created doc: WITH

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

**Comment by Amber1990Zhang** [comment]
LGTM

File: docs-2.0/3.ngql-guide/8.clauses-and-options/with.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+> **NOTE:** `WITH` only works in openCypher syntax. In nGQL syntax, use pipe symbols instead.
+
+## Combine statements and form a composite query
+
+Use a `WITH` clause to combine statements and transfer the output of a statement as the input of another statement.
+
+* Example 1:
+
+    ```ngql
+    nebula> MATCH p=(v:player{name:"Tim Duncan"})--() \
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/with.md:16]: awesome!

File: docs-2.0/3.ngql-guide/8.clauses-and-options/with.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    +----------------------------------------------------------------------+
+    | ("player108" :player{age: 36, name: "Boris Diaw"})                   |
+    +----------------------------------------------------------------------+
+    Got 12 rows (time spent 3795/4487 us)
+    ```
+
+* Example 2:
+
+    ```ngql
+    nebula> MATCH (v) WHERE id(v)=="player100" WITH labels(v) AS tags_unf UNWIND tags_unf AS tags_f RETURN tags_f;
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/with.md:53]: break lines

**Review by whitewum — DISMISSED** [review]
awesome

**Review by Amber1990Zhang — APPROVED** [review]
great job

**Review by whitewum — APPROVED** [review]

**Pull request merged**